#0:chargingPile 1:obj 2:motion
int32 cmd
---